docs/source/contribution-guidelines.rst

```diff
@@ -0,0 +1,7 @@
+=======================
+Contribution Guidelines
+=======================
+
+Guidelines about how to contribute
+
```

Comment:
I have added a fancy `todo` check it out. Also if you like we can make a separate page for To do's it can list all that stuff that we plan to do. All of it will be automatic 
https://www.sphinx-doc.org/en/master/development/tutorials/todo.html 
https://www.sphinx-doc.org/en/master/usage/extensions/todo.html

Reply:
I supposed our separate TODO page is our list of issues on GH right now. This may be related or unrelated, but as we release each new tagged version, there should be a changes page describing the features added, bugfixes etc. Although, I think this may be outside of the scope of this PR.